br/pkg/lightning/backend/local/engine.go

```diff
@@ -1561,7 +1561,7 @@ func (i dbSSTIngester) mergeSSTs(metas []*sstMeta, dir string) (*sstMeta, error)
 	heap.Init(mergeIter)
 
 	name := filepath.Join(dir, fmt.Sprintf("%s.sst", uuid.New()))
-	writer, err := newSSTWriter(name)
+	writer, err := newSSTWriter(name, 16*1024)
```

Comment:
we can also use the block size passed in by user instead of hard-code value

Reply:
this is the test code. And i couldn't access to config here

Reply:
This is not the test code, this is lightning merging pebble SST, and it should use the block size that user passes in, otherwise this PR does not have actual effect since `dbSSTIngester` is what really has effect. 
You can try to pass `BlockSize` through functions and struct to `dbSSTIngester` by adding arguments or fields, then here you can acess `BlockSize` using `dbSSTIngester`.

Reply:
oh ok. I didn't realize it. I will make these changes.

Reply:
As per the comments, 
``sstIngester is a interface used to merge and ingest SST files.
1505 // it's a interface mainly used for test convenience```

It is used only for testing. And the callers in 6.5 are only from test code. However, i see that there are additional callers in 7.5 . What is the reason of that ?

Reply:
I understand it now. In our case, rows are always ordered so compact is always set to false. That is why it worked for us. I will make the proposed change. 
```657     if !tr.tableMeta.IsRowOrdered {
 658         dataEngineCfg.Local.Compact = true
 659         dataEngineCfg.Local.CompactConcurrency = 4
 660         dataEngineCfg.Local.CompactThreshold = local.CompactionUpperThreshold
 661     }